be/src/test/java/com/jjikmuk/sikdorak/integration/user/UserCreateIntegrationTest.java

```diff
@@ -0,0 +1,36 @@
+package com.jjikmuk.sikdorak.integration.user;
+
+import com.jjikmuk.sikdorak.integration.InitIntegrationTest;
+import com.jjikmuk.sikdorak.user.domain.User;
+import com.jjikmuk.sikdorak.user.domain.UserRespository;
+import com.jjikmuk.sikdorak.user.exception.DuplicateUserException;
+import com.jjikmuk.sikdorak.user.service.UserService;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class UserCreateIntegrationTest extends InitIntegrationTest {
+
+    @Autowired
```

Comment:
테스트 클래스 최상단에 테스트 설명이 있으면 좋을거같아요!

```suggestion
import static org.assertj.core.api.Assertions.assertThat;
import static org.assertj.core.api.Assertions.assertThatThrownBy;

@DisplayName("UserCreate 통합테스트")
public class UserCreateIntegrationTest extends InitIntegrationTest {

    @Autowired
```

Reply:
👍👍👍

Reply:
단위테스트에도 마찬가지였어유 ㅜ 🙇‍♂️